### PR TITLE
Update readme and create more flushed out contributing documentation

### PR DESCRIPTION
**Pull request by corwintines**
## Description

- Update readme to be more friendly for all contributor types
- Update /contributing page with updated resources
    - https://ethereumorgwebsitedev01-readmeupdate.gatsbyjs.io/en/contributing
- Create pages for testing, code, and content contributions
    - https://ethereumorgwebsitedev01-readmeupdate.gatsbyjs.io/en/contributing/code
    - https://ethereumorgwebsitedev01-readmeupdate.gatsbyjs.io/en/contributing/content
    - https://ethereumorgwebsitedev01-readmeupdate.gatsbyjs.io/en/contributing/testing 

Fixes: https://github.com/ethereum/ethereum-org-website/issues/8524

### CODE REVIEW

**Comment by rohan9024** [comment]
Hey @corwintines ! Just noticed some extra spacing on this url --> https://ethereumorgwebsitedev01-readmeupdate.gatsbyjs.io/en/contributing/testing/#user-experience

image


1. Is there any image to be rendered which is not being rendering ? 
2. On this link--> https://ethereumorgwebsitedev01-readmeupdate.gatsbyjs.io/en/contributing/code/#1.-submit-an-issue
The links on the right side are not functional

For reference, 
https://github.com/ethereum/ethereum-org-website/assets/64919642/2424fc75-9c92-4456-a5ef-ea1041132701

_In the above video, I'm trying to click the links which should navigate me to that part._

3. Feature Request: We can link the open issues on the contributing.md itself or have a separate page for it. Additionally, in order to let the developers gain confidence, we can have simple good first issues. 

For reference: https://appwrite.io/community

4. Adding New "Require Guidance Issue Template" and also label: When I click on create new issue, there are several templates like this-->
image

We can add require guidance which can be helpful for developers who require assistance. Assistance can be related to setting up the project , problems related to codebase, structure, etc.


Lastly, I would love to work on these issues. Feel free to assign me.
Have a Great Day!😊

**Comment by konopkja** [comment]
Screenshot 2023-09-25 at 19 48 51
can this just be a link to the folder in our GH repo?
https://github.com/ethereum/ethereum-org-website/tree/dev/src/content

**Comment by konopkja** [comment]
Screenshot 2023-09-25 at 19 52 30

@corwintines  this should have been simple bullet points, not a lengthy paragraph :P

**Comment by konopkja** [comment]
Screenshot 2023-09-25 at 19 53 45

this should be under the headline, not at the end

File: src/content/contributing/testing/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Testing contributions
+description: Testing contribution to ethereum.org
+lang: en
+---
+
+# Testing contributions to ethereum.org
+
+The QA testing contributions is an initiative to test release candidates for ethereum.org before they are deployed into production. Testing is a vital part in a products release cycle to catch bugs and regressions that may have been introduced before a release.
+
+On Tuesdays we create a release candidate that goes into the staging branch. Information about the release candidate will be posted in the `👾 ｜ website-code` channel in our [Discord](https://discord.gg/ethereum-org). You can also find the release notes for the release candidate on the [releases page](https://github.com/ethereum/ethereum-org-website/releases) in the GitHub repository. The release candidate will be tagged as a `pre-release` until it has been merged into master. After QA testing, we deploy the release candidate into production if there are no release blocking bugs found on Thursday. More info on the deploy process for ethereum.org can be found [here](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/deploy-process.md).
```
Comment (konopkja) [inline src/content/contributing/testing/index.md:11]:
```suggestion
 in the GitHub repository. The release candidate will be tagged as a `pre-release` until it has been merged into master. After QA testing, we deploy the release candidate into production if there are no release blocking bugs found on Thursday. More info on the deploy process for ethereum.org can be found [here](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/deploy-process.md).

- **Every Tuesday** we create a release candidate that goes into the staging branch.
- Info about the release candidate is posted in the `👾 ｜ website-code` channel in our [Discord](https://discord.gg/ethereum-org). 
-  Find the release notes for the release candidate on the [releases page](https://github.com/ethereum/ethereum-org-website/releases)
- The release candidate will be tagged as a `pre-release`
- **On Thursday** the release candidate is published on production.
- More info on the deploy process for ethereum.org can be found [here](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/deploy-process.md).

```

File: src/content/contributing/testing/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## How to get involved
+
+There are a number of ways to get involved with testing. We are looking for any content, design, or user experience bugs or regressions that may have been introduced in the release candidate.
```
[the remark this answers is not in the document]
Comment (corwintines) [reply src/content/contributing/testing/index.md:15]: A bit. We currently don't offer anything, but I would like to see this as something we do for 2024.

**Comment by wackerow** [comment]
Just updated with latest from `dev`... Just noting this PR does _not_ address the removal of Gatsby information; this is handled in #12041

File: src/content/contributing/testing/index.md
Comment (wackerow) [inline src/content/contributing/testing/index.md:1]: This file needs custom header IDs still

File: src/content/contributing/code/index.md
Comment (wackerow) [inline src/content/contributing/code/index.md:1]: This file needs custom header IDs still

File: public/content/contributing/index.md
Comment (wackerow) [inline public/content/contributing/index.md:1]: This file needs custom header IDs still (on new items)

File: src/content/contributing/content/index.md
Comment (wackerow) [inline src/content/contributing/content/index.md:1]: Might just suggest adding the programming language to the code blocks here

File: README.md [hunk 3]
```diff
@@ -10,176 +10,41 @@
...
-- _Confirm your GC preview deploy looks & functions as expected_
-- Why not say hi and draw attention to your PR in [our discord server](https://discord.gg/ethereum-org)?
-
-### 7. Wait for review
+### Join our Discord server
 
-- The website team reviews every PR
-- See [how decisions are made on content changes](https://ethereum.org/en/contributing/#how-decisions-about-the-site-are-made)
-- Acceptable PRs will be approved & merged into the `dev` branch
+We have a space to discuss all things ethereum.org – share your ideas or just say hi over [on Discord](https://discord.gg/CetY6Y4).
```
Comment (wackerow) [inline README.md:17]:
```suggestion
We have a space to discuss all things ethereum.org – share your ideas or just say hi over [on Discord](https://discord.gg/ethereum-org).
```

File: src/content/contributing/code/index.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+### 6. Submit your PR
+
+- After your changes are committed to your GitHub fork, submit a pull request (PR) to the `dev` branch of the `ethereum/ethereum-org-website` repo
+- In your PR description, reference the issue it resolves (see [linking a pull request to an issue using a keyword](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
+  - ex. `Updates out of date content [Fixes #1234]`
+- Gatsby Cloud (our hosting service for build previews) deploys all PRs to a publicly accessible preview URL, e.g.:
+  ![Gatsby Cloud deploy preview](./GC-preview-deploy.png)
+- _Confirm your GC preview deploy looks & functions as expected_
+- Why not say hi and draw attention to your PR in [our discord server](https://discord.gg/CetY6Y4)?
```
Comment (wackerow) [inline src/content/contributing/code/index.md:120]:
```suggestion
- Why not say hi and draw attention to your PR in [our discord server](https://discord.gg/ethereum-org)?
```

**Review by wackerow — CHANGES_REQUESTED** [review]
Nice, left a few comments and suggestions

**Pull request closed without merging**